test/parallel/test-repl-import-referrer.js

```diff
@@ -8,23 +8,24 @@ const args = ['--interactive'];
 const opts = { cwd: fixtures.path('es-modules') };
 const child = cp.spawn(process.execPath, args, opts);
 
-const outputs = [];
+let output = '';
 child.stdout.setEncoding('utf8');
 child.stdout.on('data', (data) => {
-  outputs.push(data);
-  if (outputs.length === 3) {
+  output += data;
+
+  if (
+    !child.stdin.writableEnded &&
+    output.includes('[Module: null prototype] { message: \'A message\' }\n')
```

Comment:
```suggestion
    output.includes('[Module: null prototype] { message: \'A message\' }\n> ')
```

I'm not sure if it makes sense, feel free to ignore.